### ISSUE TITLE
When logging in redirecting does not work

### ISSUE THREAD

**Issue by vanlummelhuizen** [Signbank/NGT-signbank#222]
When requesting a page that needs the user to be logged in, the user gets a login page. After logging in, the user is not redirected to the originally requested page but to `registration/login.html.

**Comment by Woseseltops** [comment]
Nice :)

**Comment by ocrasborn** [comment]
Well, yes, @Woseseltops, but it doesn't yet work for ASL, or perhaps the step before doesn't: if i'm not logged in and select Signs > Search from the menu, I get an error page ("The requested URL /accounts/login/ was not found on this server.", trying to read http://applejack.science.ru.nl/accounts/login/?next=/asl-signbank/signs/search/).

**Comment by Woseseltops** [comment]
>  or perhaps the step before doesn't:

Yeah, the problem you pointed out was different: this part of Django didn't realize the ASL signbank is at a 'subfolder' url. After an hour of Googling I discovered that there is a Django setting for this, so it works now. By the way, this is a nice example of time I wouldn't have had to spend if we managed to tackle #190  :) . 

@vanlummelhuizen speaking of subfolder urls, see https://github.com/Signbank/NGT-signbank/commit/aa6d4371594a613374ffdd1643c34d969f7ce323 . There was a small change I had to do make sure your addition also worked for the ASL signbank url.
